sdk/eventhubs/azure_messaging_eventhubs/examples/eventhubs_batch_produce_events.rs

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+use azure_core::Uuid;
 /// This sample demonstrates how to send events to all partitions using a batch sender.
 ///
 use azure_identity::DefaultAzureCredential;
-use azure_core::Uuid;
 use azure_messaging_eventhubs::{models::EventData, EventDataBatchOptions, ProducerClient};
```

Comment:
Nits: was that intentional to move the import before the `///` comment? Also, while it won't do much for this *example*, module docs are `//!` comments.

Reply:
No, that was the Rust analyzer being profoundly stupid w.r.t. where it auto-adds `use` statements.

I usually just fix them but I missed this one.